notebooks/DM-31046-Zooniverse-cutouts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DM-31046 Zooniverse cutouts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is to try out the `ZoonverseCutoutsTask` implemented on DM-31046. The notebook may evolve as that task is improved (i.e. to include source selection via sql query) on a subsequent ticket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import sqlite3\n",
+    "\n",
+    "import IPython.display\n",
+    "\n",
+    "import lsst.daf.persistence\n",
+    "from lsst.pipe.tasks.zooniverseCutouts import ZooniverseCutoutsTask\n",
+    "\n",
+    "# local to ap_pipe-notebooks\n",
+    "import apdbPlots"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The gen2 butler repos containing the templates and difference images, respectively.\n",
+    "parent = \"/project/sullivan/saha/cwcoadds_processed\"\n",
+    "REPO = '/project/sullivan/saha/ap_pipe/cw/'"
```

Comment:
Can't you use repo chaining here? I would have expected searching `REPO` to find the templates if the difference images were generated from them.